src/dmd/root/file.d

```diff
@@ -78,12 +73,6 @@ struct File
 
 nothrow:
     /// Read the full content of a file.
-    extern (C++) static ReadResult read(const(char)* name)
```

Comment:
Weird that it's not symmetric to write, maybe better keep both overloads of just the D char array one.
https://dev.azure.com/dlanguage/dmd/_build/results?buildId=29048&view=logs&j=6775e4b2-0d76-5db7-9bcd-015c7821c824&t=1bf40372-f529-586c-b834-12b53d9adf74&l=343